sql/core/src/main/scala/org/apache/spark/sql/execution/Generate.scala

```diff
@@ -34,24 +35,12 @@ import org.apache.spark.sql.catalyst.expressions._
 @DeveloperApi
 case class Generate(
     generator: Generator,
+    output: Seq[Attribute],
```

Comment:
I agree, the deletion of L42-L54 is duplicated code with the `logical.Generate`, I would like to keep it unchange for this PR, which aims to fix the bug.
Definitely we need to refactor the code of `Generator` and `Generate`, but we can leave it after 1.3 release, what do you think?